### PR TITLE
Task - Footer

### PR DESCRIPTION
**Pull request by yonikid15**
# Notes
- Adds Footer component to Dashboard

### CODE REVIEW

File: admin/resources/js/components/Footer.tsx
Comment (tristan-orourke) [inline admin/resources/js/components/Footer.tsx:143]: You can do this already with ``href=`https://www.canada.ca/${intl.locale}.html` ``. It'll always be en right now, of course, but it'll be future-proofed.

File: admin/resources/css/app.css
Comment (tristan-orourke) [inline admin/resources/css/app.css:42]: H2 has a container attribute. Its different from this one? https://hydrogen.design/#containers

**Review by tristan-orourke — CHANGES_REQUESTED** [review]
Besides the localized link, there's a little bar of white beneath the footer that's not supposed to be there, I think. 

image

File: admin/resources/css/app.css
Comment (yonikid15) [reply admin/resources/css/app.css:42]: Yea this is different. I used it to stack the content and footer correctly. Also, it should be easy to add the goc banner and header as well with this setup.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:40]: Change canada to Canada.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:45]: Change canada to Canada.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:56]: Change canada to Canada.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:61]: Change canada to Canada.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:66]: Change canada to Canada.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:70]: What's the purpose of `<{}>`?

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:79]: Could this be dynamic and use `${intl.locale}` instead of `en`?

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:84]: Could this be dynamic and use `${intl.locale}` instead of `en`?

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:108]: Is there an alternative to `gap` since it's a [newer property](https://caniuse.com/?search=gap)?

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:149]: This path doesn't work in Storybook.

**Review by mnigh — CHANGES_REQUESTED** [review]
Nice work. I have added some comments and requested some changes that need to be resolved before approval though.

File: admin/resources/js/components/Footer.tsx
Comment (yonikid15) [reply admin/resources/js/components/Footer.tsx:108]: I don't think we are supporting ie11 anymore, so this should be fine.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:142]: The URL should be lowercase _canada_ not _Canada_.

File: admin/resources/js/components/Footer.tsx
Comment (mnigh) [inline admin/resources/js/components/Footer.tsx:89]: The URL should be lowercase _canada_ not _Canada_.

**Review by mnigh — CHANGES_REQUESTED** [review]

**Review by mnigh — APPROVED** [review]
👍

**Pull request closed without merging**
